tests/test_matrix.py

```diff
@@ -126,6 +127,37 @@ def test_cholesky_matrix_decomposition(self):
                              [[5, 1.2, 0.3, -0.6], [1.2, 6, -0.4, 0.9],
                               [0.3, -0.4, 8, 1.7], [-0.6, 0.9, 1.7, 10]]))
 
+class TestInversion(unittest.TestCase):
+    """[summary]
+    Test for the file multiply.py
```

Comment:
Code looks good, just the test is for the file matrix_inversion.py

Reply:
I forgot to change it when I was copying the format, my bad! Fixed now. :)